sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/implementation/models/TeamsUserExchangeTokenRequest.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.identity.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The TeamsUserExchangeTokenRequest model. */
+@Fluent
+public final class TeamsUserExchangeTokenRequest {
```

Comment:
Shouldn't this class be removed?

Reply:
yes, totally forgot to delete it, thanks